docs/synapse/userguides/ug015_storm_ref_pivot.rst

```diff
@@ -33,6 +33,52 @@ jointags()
 ----------
 Todo
 
+tree()
+------
+
+The `tree()` operator acts as a recursive pivot for forms which reference their own types or multiple duplicate ptypes.
+
+**Operator Syntax:**
+
+.. parsed-literal::
+
+  *<query>* **tree(** *<srcprop> , <dstprop>,* [ *recurnlim=<n>* ] **)**
+
+  *<query>* **tree(** *<relative srcprop>, <dstprop>,* [ *recurnlim=<n>* ] **)**
+
+  *<query>* **tree(** *<dstprop>,* [ *recurnlim=<n>* ] **)**
+
+**Macro Syntax:**
+
+There is no macro syntax for the tree() operator.
+
+**Examples:**
+::
+
+  # Full form - traversing to all of the woot.com children nodes
+  inet:fqdn = woot.com tree( inet:fqdn, inet:fqdn:domain )
+
+  # Relative source only form - traversing ou:suborg relationships
+  ou:org:alias = someorg -> ou:suborg:org tree( :sub, ou:suborg:org ) :sub -> ou:org
+
+  # Destination only form - traversing all of the woot.com children nodes.
+  inet:fqdn = woot.com tree( inet:fqdn:domain )
+
+  # Select the entire syn:tage=foo tree.
+  syn:tag=foo tree(syn:tag, syn:tag:up)
+
+  # tree() up - select all parent fqdns of mx.somebox.woot.com
+  inet:fqdn = mx.somebox.woot.com tree( inet:fqdn:domain, inet:fqdn )
+
```

Comment:
For consistency with other docs, consider adding a bold header for:

**Usage Notes**

After examples and before your "notes".
Similarly, consider making individual notes bullet points.

Really just a formatting nitpick, it's good either way!

Reply:
fixed these up! thanks.